### DISCUSSION TITLE
ROBB ROB_200-017-0 not showing energy

### DISCUSSION THREAD

**Discussion by dromer1967** [Koenkk/zigbee2mqtt#14469] [Devices]
I have purchased a ROBB ROB_200-017-0 smart plug. After pairing it, it shows the voltage, current, power but not the energy.

The zigbee2mqtt documentation states that it does support energy.

Does anyone know what I am doing wrong and how I can enable the device in zigbee2mqtt so it does show energy consumption?

I have re-paired it more than once and also have upgraded to the latest zigbee2mqtt release, but that didn't help.

I bought the device for the energy consumption reading, so I hope someone can help me here :)

Regards,
Peter

**Comment by nielsvanbaal** [comment]
I have the same problem :(

**Comment by GrasshopperNL** [comment]
Same here

**Comment by Koenkk** [comment]
Please check https://github.com/Koenkk/zigbee2mqtt/issues/14471

**Comment by dromer1967** [comment]
I have got it working now! I put the solution in issue #14471
